packages/api/package.json

```diff
@@ -31,23 +31,20 @@
     "check-readme": "typescript-docs-verifier"
   },
   "dependencies": {
-    "types": "*",
-    "db": "*",
-    "executor": "*",
-    "body-parser": "1.20.1",
     "class-transformer": "0.5.1",
     "class-validator": "0.14.0",
-    "compression": "1.7.4",
+    "db": "*",
     "ethers": "5.7.2",
-    "express": "4.18.2",
-    "joi": "17.7.0",
+    "executor": "*",
+    "fastify": "^4.14.1",
```

Comment:
Pls use a specific version of the package.

Reply:
done